server/routes/printReport/NoticeOfBeingPutOnReportData.ts

```diff
@@ -0,0 +1,36 @@
+import { ConfirmedOnReportData } from '../../data/ConfirmedOnReportData'
+import { convertToTitleCase, formatTimestampTo } from '../../utils/utils'
+
+export default class NoticeOfBeingPutOnReportData {
```

Comment:
I've used an object to pass about the template because there were a lot of properties and it was easy to get the ordering wrong, and its used in 2 places so generating the properties in one place made sense